conf/messages

```diff
@@ -1,35 +1,44 @@
 # https://www.playframework.com/documentation/latest/ScalaI18N
 edit-set.heading=Edit set: {0}
 edit-set.record.discard.button=Discard changes
+edit-set.record.discard.subheader=Changes discarded
 edit-set.record.discard.text=Changes to the following record have been discarded:
+edit-set.record.edit.ccr=CCR
 edit-set.record.edit.covering-dates=Covering dates
 edit-set.record.edit.end-date=End date
 edit-set.record.edit.former-reference=Former reference - Department (optional)
 edit-set.record.edit.heading=TNA reference: {0}
 edit-set.record.edit.id=PAC-ID: {0}
-edit-set.record.edit.scope-and-content=Scope and content
 edit-set.record.edit.legend=Intellectual properties
+edit-set.record.edit.scope-and-content=Scope and content
 edit-set.record.edit.start-date=Start date
 edit-set.record.edit.title=Edit record
+edit-set.record.error.covering-dates-too-long=Covering date too long, maximum length 255 characters
+edit-set.record.error.covering-dates=Covering date format is not valid
 edit-set.record.error.former-reference-department=Former reference - Department too long, maximum length 255 characters
 edit-set.record.error.scope-and-content=Scope and content too long, maximum length 8000 characters
-edit-set.record.error.covering-dates=Covering date format is not valid
-edit-set.record.error.covering-dates-too-long=Covering date too long, maximum length 255 characters
 edit-set.record.missing.covering-dates=Enter the covering dates
 edit-set.record.missing.scope-and-content=Enter the scope and content
+edit-set.record.save.back=Back to edit record
 edit-set.record.save.button=Save changes
-edit-set.record.save.confirmation=Your changes have been saved.
+edit-set.record.save.subheader=Changes saved
 edit-set.record.save.text=Changes to the following record have been saved:
 edit-set.title=Edit set
-header.title=Pan-Archival Catalogue
+error.summary.title=There is a problem
 header.logout=Sign out
+header.title=Pan-Archival Catalogue
 index.heading=Welcome to the Catalogue
 index.title=Catalogue
 login.authentication.error=Username and/or password is incorrect.
+login.authentication.problem=There is a problem
 login.button.text=Sign in
 login.heading=Sign in: Pan-Archival Catalogue
 login.missing.password=Enter a password
 login.missing.username=Enter a username
 login.password.label=Password
 login.title=Sign in
 login.username.label=Username
+
+placeholder.back-to-coal-80=Back to edit set (COAL 80 Sample)
```

Comment:
Added these placeholders into messages as well, I think it will make them easier to delete

Reply:
I think the edit-set-specific bit (e.g. "COAL 80 Sample") could be passed into the view. It's just the edit set name, after all.

Reply:
OK.